### PR TITLE
Leviathan Dreadnought

### PR DESCRIPTION
**Pull request by revivedtitan**
Added/Updated Leviathon Dreadnought

> ⚠️  NOTE: use notes like this to emphasize something about the PR. This could
> include other PRs this PR is built on top of; reasons for why the PR is on hold; 
or anything else you would like > to draw attention to

## Description
_A few sentences describing the overall goals of the pull request.
What is the current behavior of the app? What is the updated/expected behavior
with this PR?_

## Related Issues
* closes # [insert issue name here](insert link here)

## Todos
_Fill out or remove list items as needed_
- [ ] Tests

## Test Case
_Notes This should be a link to test roster file that evidences your change._
It would also be acceptable to upload the file as a comment.

### CODE REVIEW

**Comment by nstephenh** [comment]
Please set the title correctly in future pull requests.

**Comment by revivedtitan** [comment]
> Please set the title correctly in future pull requests.

In the future do you just want the unit added as the description/title? I didn't see any guidelines for naming and want to make sure it's good next time.

File: 2022 - Horus Heresy.gst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<gameSystem id="28d4-bd2e-4858-ece6" name="(HH V2) Horus Heresy (2022)" revision="6" battleScribeVersion="2.03" xmlns="http://www.battlescribe.net/schema/gameSystemSchema">
+<gameSystem id="28d4-bd2e-4858-ece6" name="(HH V2) Horus Heresy (2022)" revision="7" battleScribeVersion="2.03" xmlns="http://www.battlescribe.net/schema/gameSystemSchema">
```
Comment (nstephenh) [inline 2022 - Horus Heresy.gst:2]: We don't need to update the revision on this file for this change. It's not a huge deal though.

File: 2022 - Legiones Astartes.cat [hunk 2]
```diff
@@ -24393,26 +24393,70 @@ A Perfect Fury: If a model or models with the Skill Unmatched special rule selec
...
+                <characteristic name="I" typeId="62d3-22d7-2d49-52dc">4</characteristic>
+                <characteristic name="A" typeId="f111-2ce5-dd12-d6b0">5</characteristic>
+                <characteristic name="Ld" typeId="e8a6-1da9-d384-8727">9</characteristic>
+                <characteristic name="Save" typeId="e593-6b3c-f169-04f0">2</characteristic>
+              </characteristics>
+            </profile>
+          </profiles>
+          <selectionEntries>
+            <selectionEntry id="a3ad-3b70-433b-b9fa" name="Atomantic Deflector" hidden="false" collective="false" import="true" type="upgrade">
+              <profiles>
```
Comment (nstephenh) [inline 2022 - Legiones Astartes.cat:24443]: Automantic deflector is not an option. This should either be an info link to wargear or a link to a shared selection entry with min1 max1.

**Review by nstephenh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**